packages/util-utf8/src/fromUtf8.browser.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { fromUtf8 } from "./fromUtf8.browser";
```

Comment:
Explicitly call out jsdom environment in browser tests
```suggestion
/**
 * @jest-environment jsdom
 */
import { fromUtf8 } from "./fromUtf8.browser";
```

Reply:
fixed